dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/ConverterService.java

```diff
@@ -86,6 +93,13 @@
         M transformedModel = projection.transformModel(modelObject);
         DSpaceConverter<M, R> converter = requireConverter(modelObject.getClass());
         R restObject = converter.convert(transformedModel, projection);
+        if (restObject instanceof BaseObjectRest) {
+            if (!dSpacePermissionEvaluator.hasPermission(SecurityContextHolder.getContext().getAuthentication(),
+                                                         restObject, "READ")) {
+                log.info("Access denied on " + restObject.getClass());
```

Comment:
I'm not sure about this log message.  Do we really want an `info` message for every access denied on every object?  Also, this looks to just print out the class name (like "ItemRest") instead of any useful info.  I think we should either enhance the message here, or consider removing it.

Reply:
I'm still not sure this `log.info` message is necessary.  It doesn't seem to provide any useful information.  Should we just remove it?

Reply:
It has been adjusted to log.debug, with the ID as well
This would still allow someone to verify the problem if some data is not embedded and it's not clear why (so they can see it's due to an access restriction)